js/w3c/headers.js

```diff
@@ -105,7 +105,7 @@ define(
     function (hb, utils, headersTmpl, sotdTmpl, cgbgHeadersTmpl, cgbgSotdTmpl, wsHeadersTmpl) {
         Handlebars.registerHelper("showPeople", function (name, items) {
             // stuff to handle RDFa
-            var re = "", rp = "", rm = "", rn = "", rwu = "", rpu = "", bn = "",
+            var re = "", rp = "", rm = "", rn = "", rwu = "", rpu = "", bn = "", rpe = "",
```

Comment:
Please use long form instead of "rpe". Not sure why the others are not in long form (bad form!).